string sphero_name
geometry_msgs/Pose2D goal

